vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/FilteringIT.java

```diff
@@ -30,13 +30,17 @@
 public class FilteringIT extends AbstractComboBoxIT {
 
     private ComboBoxElement box;
+    private ComboBoxElement comboBoxWithFilteredItems;
 
     @Before
     public void init() {
         open();
         waitUntil(driver -> findElements(By.tagName("vaadin-combo-box"))
                 .size() > 0);
         box = $(ComboBoxElement.class).first();
+
+        comboBoxWithFilteredItems = $(ComboBoxElement.class)
+                .id("combo-box-with-filtered-items");
```

Comment:
Use the package-private constant proposed to be defined in the `FilteringPage.java`.

Reply:
done